src/components/OptionRow.js

```diff
@@ -174,6 +174,10 @@ class OptionRow extends Component {
                                 this.props.isDisabled && styles.cursorDisabled,
```

Comment:
I know it wasn't changed in your PR, but this style is now redundant as `PressableWithFeedback` is managing it itself. Thus this line can be removed.